modulefiles/gsi_jet.lua

```diff
@@ -1,24 +1,23 @@
 help([[
 ]])
 
-prepend_path("MODULEPATH", "/mnt/lfs4/HFIP/hfv3gfs/role.epic/hpc-stack/libs/intel-18.0.5.274/modulefiles/stack")
+prepend_path("MODULEPATH", "/mnt/lfs4/HFIP/hfv3gfs/role.epic/miniconda3/modulefiles")
```

Comment:
Does this `prepend_path` point at the official installation all UFS and EMC apps should use?

Reply:
@natalie-perlin Confirmed this is the correct miniconda3 installation.

Reply:
Resolve with caveat noted above that g-w and library team staff need to confirm that machine specific modulefiles in this PR use officially maintained and supported modules.